tests/platform_helper/test_exceptions.py

```diff
@@ -76,8 +76,8 @@
         ),
         (
             ImageNotFoundException,
-            {"commit": "test-commit-hash"},
-            """The commit hash "test-commit-hash" has not been built into an image, try the `platform-helper codebase build` command first.""",
+            {"image_ref": "abc1234"},
+            """An image labelled "abc1234" could not be found in your image repository. Try the `platform-helper codebase build` command first.""",
```

Comment:
To express the intent...
```suggestion
            {"image_ref": "does-not-exist"},
            """An image labelled "does-not-exist" could not be found in your image repository. Try the `platform-helper codebase build` command first.""",
```